src/DotNetNuke.PowerBI/scripts/ContentView.js

```diff
@@ -56,13 +56,24 @@
             accessToken: context.Token,
             embedUrl: context.EmbedUrl,
             id: context.Id,
-            permissions: that.models.Permissions.Read,
+            permissions: that.models.Permissions.All,
```

Comment:
The permissions should depend on user's permissions. i.e.:

permissions: context.hasEditPermissions ? that.models.Permissions.All : that.models.Permissions.Read,